srcs/runners_manager/runner/Manager.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, settings: dict,
     def remove_all_runners(self):
         for manager in self.runner_managers:
             for runner in manager.get_runners().values():
-                manager.delete_runner(runner)
+                if not runner.is_running:
+                    manager.delete_runner(runner)
```

Comment:
so now when we reset, we don't delete runners that have a job ongoing?

Reply:
By default github won't allow them to be deleted.

Reply:
true

Reply:
BUT ! we can consider that if we use this function, we want to try in the case our database is not synchronized with github

Reply:
Yes, at first, for this scenario I was actually gonna suggest having a soft reset and a hard reset, but if we can't delete a runner that is running the hard reset won't work.

So all good here, this function will actually be safe to use at any time so that's good.